<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="1" hostname="LAPTOP-AJDG01Q7" failures="2" tests="2" name="Default test(failed)" time="0.129" errors="0" timestamp="2025-09-12T09:09:57 IST">
  <testcase classname="TestNGC.First.RetryAnalyzerClass" name="testcase02" time="0.028">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at TestNGC.First.RetryAnalyzerClass.testcase02(RetryAnalyzerClass.java:17)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- testcase02 -->
  <testcase classname="TestNGC.First.RetryAnalyzerClass" name="testcase01" time="0.003">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at TestNGC.First.RetryAnalyzerClass.testcase01(RetryAnalyzerClass.java:11)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- testcase01 -->
  <testcase classname="TestNGC.First.RetryAnalyzerClass" name="testcase03" time="0.0">
    <ignored/>
  </testcase> <!-- testcase03 -->
</testsuite> <!-- Default test(failed) -->
